[2025-07-28T18:27:33.334+0000] {executor_loader.py:235} INFO - Loaded executor: LocalExecutor
[2025-07-28T18:27:34.029+0000] {logging_mixin.py:188} INFO - Sending to executor.
[2025-07-28T18:27:34.172+0000] {base_executor.py:149} INFO - Adding to queue: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:27:34.208+0000] {local_executor.py:403} INFO - Shutting down LocalExecutor; waiting for running tasks to finish.  Signal again if you don't want to wait.
[2025-07-28T18:27:34.206+0000] {local_executor.py:90} INFO - QueuedLocalWorker running ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:27:34.378+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py:110 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T18:27:34.503+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/***/dags/financial_etl_dag.py
[2025-07-28T18:27:34.656+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]> on host 69f65e32f51d
[2025-07-28T18:27:34.780+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T18:27:35.012+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=1, removed=0, done=1, success_setup=0, skipped_setup=0), upstream_task_ids={'load_data'}
[2025-07-28T18:27:35.013+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Task Instance State' FAILED: Task is in the 'upstream_failed' state.
[2025-07-28T18:27:35.015+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
[2025-07-28T18:45:39.780+0000] {executor_loader.py:235} INFO - Loaded executor: LocalExecutor
[2025-07-28T18:45:40.440+0000] {logging_mixin.py:188} INFO - Sending to executor.
[2025-07-28T18:45:40.518+0000] {base_executor.py:149} INFO - Adding to queue: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:45:40.584+0000] {local_executor.py:403} INFO - Shutting down LocalExecutor; waiting for running tasks to finish.  Signal again if you don't want to wait.
[2025-07-28T18:45:40.595+0000] {local_executor.py:90} INFO - QueuedLocalWorker running ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:45:40.687+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py:110 DeprecationWarning: This process (pid=60) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T18:45:40.714+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/***/dags/financial_etl_dag.py
[2025-07-28T18:45:40.916+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]> on host 502279ba8596
[2025-07-28T18:45:41.021+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T18:45:41.192+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Task Instance State' FAILED: Task is in the 'upstream_failed' state.
[2025-07-28T18:45:41.200+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=1, removed=0, done=1, success_setup=0, skipped_setup=0), upstream_task_ids={'load_data'}
[2025-07-28T18:45:41.204+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
[2025-07-28T18:46:36.549+0000] {executor_loader.py:235} INFO - Loaded executor: LocalExecutor
[2025-07-28T18:46:36.739+0000] {logging_mixin.py:188} INFO - Sending to executor.
[2025-07-28T18:46:36.741+0000] {base_executor.py:149} INFO - Adding to queue: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:46:36.745+0000] {local_executor.py:403} INFO - Shutting down LocalExecutor; waiting for running tasks to finish.  Signal again if you don't want to wait.
[2025-07-28T18:46:36.745+0000] {local_executor.py:90} INFO - QueuedLocalWorker running ['***', 'tasks', 'run', 'financial_etl_pipeline', 'aggregate_monthly_summary', 'manual__2025-07-28T17:43:49.228925+00:00', '--local', '--pool', 'default_pool', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py']
[2025-07-28T18:46:36.758+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py:110 DeprecationWarning: This process (pid=282) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T18:46:36.771+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/***/dags/financial_etl_dag.py
[2025-07-28T18:46:36.852+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]> on host 502279ba8596
[2025-07-28T18:46:36.904+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T18:46:36.928+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=0, skipped=0, failed=0, upstream_failed=1, removed=0, done=1, success_setup=0, skipped_setup=0), upstream_task_ids={'load_data'}
[2025-07-28T18:46:36.929+0000] {taskinstance.py:2063} INFO - Dependencies not met for <TaskInstance: financial_etl_pipeline.aggregate_monthly_summary manual__2025-07-28T17:43:49.228925+00:00 [upstream_failed]>, dependency 'Task Instance State' FAILED: Task is in the 'upstream_failed' state.
[2025-07-28T18:46:36.931+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
